### DISCUSSION TITLE
ODBC External Table，sqlserver如何解决SSL问题

### DISCUSSION THREAD

**Discussion by ztyd** [apache/doris#8432] [A - General / Q&A]
使用odbc外表链接sqlserver，如何配置链接参数，如“"Encrypt=Yes; TrustServerCertificate=no;"”，配置链接不需要SSL认证，默认需要SSL认证，SSL Provider: [error:80001044:lib(128):func(1):internal error:unexpected error] 
配置参数没有地方配置，能提供教程文章吗，缺少sqlserver的例子

**Comment by hf200012** [comment]
This can only be checked in the official Microsoft documentation
You add me on WeChat: 35926237, I will help you solve it
